lib/registration.pm

```diff
@@ -934,7 +934,7 @@ sub scc_deregistration {
         # Remove registercloudguest when use suseconnect in a non public cloud environment.
         if (get_var('SCC_ADDONS', '') =~ /pcm/) {
             my $cloudguest_bin = "/usr/sbin/registercloudguest";
-            assert_script_run "test -f $cloudguest_bin && rm -f $cloudguest_bin";
+            script_run "[ -f $cloudguest_bin ] && rm -f $cloudguest_bin || true";
```

Comment:
I think you don need return true when not found, script run won't check return value.

Reply:
Now rerunning all VRs after OSD is back.